.chloggen/drop-resource-attribute-from-span.yaml

```diff
@@ -0,0 +1,30 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Since this is a breaking change, I think we should have it protected with a feature gate to give users some time to change what they have.